casepro/profiles/models.py

```diff
@@ -369,5 +369,10 @@ def _user_as_json(user, full=True, org=None):
 User.can_manage = _user_can_manage
 User.can_edit = _user_can_edit
 User.remove_from_org = _user_remove_from_org
-User.__unicode__ = _user_unicode
 User.as_json = _user_as_json
+
+
+if six.PY2:
```

Comment:
This class uses the `@python_2_unicode_compatible` decorator so we have to money patch different methods depending on Python version.